src/adapters/peggedAssets/grai/index.ts

```diff
@@ -24,78 +9,13 @@ const chainContracts: ChainContracts = {
     bridgedFromETH: ["0x894134a25a5faC1c2C26F1d8fBf05111a3CB9487"],
   },
   era: {
-    bridgedFromETH: ["0x894134a25a5faC1c2C26F1d8fBf05111a3CB9487"],
+    bridgedFromETH: ["0x5FC44E95eaa48F9eB84Be17bd3aC66B6A82Af709"],
```

Comment:
?

Reply:
yeah, think it is incorrect: https://explorer.zksync.io/address/0x894134a25a5faC1c2C26F1d8fBf05111a3CB9487

correct one: https://explorer.zksync.io/address/0x5FC44E95eaa48F9eB84Be17bd3aC66B6A82Af709#contract